docs/configuration/logging.md

```diff
@@ -35,8 +35,10 @@ To collect logs from your Codacy installation using fluentd change `fluentdopera
 
 The fluentd daemonset will send the logs to minio which is also installed by this chart.
 
-In order to send them to our support in case of problems, run the following command locally (replacing the `<namespace>` with the namespace in which Codacy was installed):
+To extract the logs and send them to Codacy's support team in case of problems, you can run the following command locally (replacing the `<namespace>` with the namespace in which Codacy was installed):
```

Comment:
Can you put a note here saying that you should create an alias for microk8s? 
Like this: https://github.com/codacy/chart/blob/master/docs/infrastructure/microk8s-quickstart.md#5-installing-codacy

Reply:
That is a good point. I prefer to deal with that in the script.
I will add a comment above.